src/components/Task.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Card, Button, CardContent, Typography, CardActions } from '@material-ui/core';
+import { TaskProps } from '../utils/Interfaces';
```

Comment:
I suggest creating a file `components/Todo/typings.ts` to add `TodoProps` interface.

Reply:
Resolved on 8df17e1b6de573c8336701b732a51a6e6bbbb122